mirdata/download_utils.py

```diff
@@ -132,7 +132,7 @@ def download_from_remote(remote, save_dir, force_overwrite=False):
                     reporthook=t.update_to,
                     data=None,
                 )
-            except Exception as e:
+            except (urllib.error.URLError, urllib.error.HTTPError) as e:
```

Comment:
could you add a comment explaining why you do this?
because errors may occur for instance when the hard drive is full it may be better to use OSError (more generic)